app/components/modals/cropper-modal.js

```diff
@@ -3,17 +3,31 @@ import ModalBase from 'open-event-frontend/components/modals/modal-base';
 export default ModalBase.extend({
   onVisible() {
     this.$('.content').css('height', '300px');
-    this.$('img').croppie({
-      customClass : 'croppie',
-      viewport    : {
-        width  : 300,
-        height : 150,
-        type   : 'square'
-      },
-      boundary: {
-        height: 250
-      }
-    });
+    if (this.get('aspectRatio')[0] === 1 && this.get('aspectRatio')[1] === 1) {
```

Comment:
Don't hard code it like this... The aspect ratio coming it could be anything .. Not just `1:1` or `2:1` ... Calculate the viewport width and height based on in. 

If there is no `aspectRatio`, default should be 2:1